modules/cmdb/services/approve/approve.go

```diff
@@ -62,7 +61,7 @@ func WithDBClient(db *dao.DBClient) Option {
 }
 
 // WithUCClient 配置 uc client
```

Comment:
Do not use chinglish

Reply:
I guess what you mean is that there should be no chinese text. and,

that's old stuff, leave it here, we have a plan to refactor all the code. until then, all chinese text will be deleted.